lib/pbench/server/api/resources/upload_api.py

```diff
@@ -63,10 +63,10 @@ def __str__(self) -> str:
 
 class Upload(ApiBase):
     """
-    Upload a dataset from an agent. This API accepts a tarball, controller
-    name, and MD5 value from a client. After validation, it creates a new
-    Dataset DB row describing the dataset, along with some metadata, and it
-    creates a pair of files (tarball and MD5 file) within the designated
+    Upload a dataset from an agent. This API accepts a tarball and MD5
+    value from a client. After validation, it creates a new Dataset DB
+    row describing the dataset, along with some metadata, and it creates
+    a pair of files (tarball and MD5 file) within the designated
     controller directory under the configured ARCHIVE file tree.
```

Comment:
> "it creates a pair of files (tarball and MD5 file) within the designated controller directory under the configured ARCHIVE file tree"

Hmm. Technically, it doesn't, and this text outlived its usefulness (and accuracy) a long time ago; that's a function of the cache manager, and shouldn't really be described here. Upload creates temporary files and hands them off to the cache manager "to do what it does". While it's not a big deal now, ideally when the cache manager stuffs the tarball into S3 we shouldn't need to change comments *here* to remain accurate.